### PR TITLE
bring fix-1 from fork into main

### PR DESCRIPTION
**Pull request by jeffschulzusc**
- sigin-in callback via angular routing
- silent callback use local oidc-client-ts
- fixed environment.prod.ts 'implicit' to 'interactive.public'

### CODE REVIEW

File: src/silent-callback.html [hunk 1]
```diff
@@ -0,0 +1,7 @@
+<script src="./oidc-client-ts.js"></script>
```
Comment (pamapa) [inline src/silent-callback.html:1]: i do not see oidc-client-ts.js part of this merge request. can you please add or adapt.

Comment (jeffschulzusc) [reply src/silent-callback.html:1]: it is there no ?
https://github.com/jeffschulzusc/sample-angular-oidc-client-ts/blob/fix-1/angular.json
https://github.com/jeffschulzusc/sample-angular-oidc-client-ts/blob/fix-1/src/silent-callback.html

Comment (pamapa) [reply src/silent-callback.html:1]: ah yes, that is the trick with the angular.json

**Review by pamapa — APPROVED** [review]

**Comment by pamapa** [comment]
thanks for contributing!

**Pull request merged**